### PR TITLE
Move projects/folders modal [SCI-5334]

### PR DESCRIPTION
**Pull request by urbanrotnik**
Jira ticket: [SCI-5334](https://biosistemika.atlassian.net/browse/SCI-5334)

### What was done
Moving projcets and folders modal

### Note
Opened questions: @aignatov-bio 
- line height

### CODE REVIEW

File: app/assets/javascripts/projects/index.js [hunk 1]
```diff
@@ -478,9 +480,90 @@
...
+
+        $(moveToModal).find('form')
+          .on('ajax:before', function() {
+            $('<input>').attr({ type: 'hidden',
+              name: 'destination_folder_id',
+              value: destinationFolder}).appendTo($(moveToModal).find('form'));
+
+            $('<input>').attr({ type: 'hidden',
+              name: 'movables',
+              value: JSON.stringify(movables)}).appendTo($(moveToModal).find('form'));
```
Comment (aignatov-bio) [inline app/assets/javascripts/projects/index.js:539]: try shorter syntax
```
$('<input>', { type: 'hidden', name: param.name, value: param.value})
```

File: app/assets/javascripts/projects/index.js [hunk 2]
```diff
@@ -478,9 +480,90 @@
...
+      projects = selectedProjects.map(e => ({id: e, type: 'project'}));
+      folders = selectedProjectFolders.map(e => ({id: e, type: 'project_folder'}));
+      let movables = projects.concat(folders);
+
+      $.get(url, {items: items}, function(result) {
+        $(moveToModal).find('.modal-content').html(result.html);
+        $(moveToModal).modal('show');
+        initializeJSTree($(moveToModal).find("#moveToFolders"));
+
+        $(moveToModal).find('form')
```
Comment (aignatov-bio) [inline app/assets/javascripts/projects/index.js:531]: save form to variable and reuse

File: app/views/projects/index/_toolbar.html.erb [hunk 3]
```diff
@@ -9,6 +9,7 @@
           <span class="fas fa-plus" aria-hidden="true"></span>
           <span class="hidden-xs"><%= t('projects.index.new') %></span>
         <% end %>
+<<<<<<< HEAD
```
Comment (aignatov-bio) [inline app/views/projects/index/_toolbar.html.erb:12]: ?

**Review by aignatov-bio — COMMENTED** [review]
Fix Hound errors

File: app/assets/javascripts/projects/index.js (hunk 1 above)
Comment (urbanrotnik) [reply app/assets/javascripts/projects/index.js:539]: for some reason, this is not working... ?

**Review by aignatov-bio — APPROVED** [review]

**Pull request merged**